index.d.ts

```diff
@@ -176,6 +220,48 @@ declare module 'plaid' {
     transaction_type: string | null;
   }
 
+  interface AssetReport {
+    asset_report_id: string;
+    client_report_id: string | null;
+    date_generated: Iso8601DateTimeString;
+    days_requested: number;
+    items: Array<AssetReportItem>;
+    user: AssetReportUser;
+  }
+
+  interface AssetReportItem {
+    item_id: string;
+    accounts: Array<AssetReportAccount>;
+    institution_id: string;
+    institution_name: string;
+    date_last_updated: string;
```

Comment:
should `date_last_updated` be one of the date types?

Reply:
⚡️